hw/ip/sysrst_ctrl/data/sysrst_ctrl_testplan.hjson

```diff
@@ -41,15 +41,15 @@
       name: combo_detect_ec_rst
       desc: '''
             Verify the combo detection with ec_rst action.
-                                     
-            * Trigger the input keys by configuring COM_SEL_CTL_0 register.                                            
-            * Set the combo duration via the COM_DET_CTL_0 register.                                                   
-            * Select the action to be taken by configuring COM_OUT_CTL_0 register.                                     
-            * Set the pulse width via EC_RST_CTL register only to raise ec_rst action.                                 
-            * Read the COMBO_INTR_STATUS register to check if the interrupt is raised and 
-              clear the interrupt. 
-            * NOTE: This is a directed test with no random values for V1 milestone, 
-              further this test will be randomized. 
```

Comment:
nit: looks like these are unrelated formatting changes. would be great to have them in a separate commit (just to be able to tell them appart). no big deal for this PR, though.

Reply:
oh I see. yeah nevermind then.